project/OsgiCheck.scala

```diff
@@ -83,7 +83,7 @@ object OsgiCheckPlugin extends AutoPlugin {
         // should be a list of one.
         val packageOwnersMap = packageOwnerTuples
           .groupBy { case (packageName, _) => packageName }
-          .mapValues { list => list.map { case (_, packageOwner) => packageOwner } }
+          .map { case (k, list) => k -> list.map { case (_, packageOwner) => packageOwner } }
```

Comment:
Thoughts on changing this to something like:
```val packageOwnersMap = packageOwnerTuples
          .groupBy { case (packageName, _) => packageName }
          .values.map { list => list.map { case (_, packageOwner) => packageOwner } }
```

I don't have a strong opinion, but I do think it makes it a little clearer that we are really only interested in the values of the map.

Reply:
I probably wouldn't have spotted this if Josh hadn't already highlighted this.

But.... anyway I can't get any intuition about now this map, nor the prior mapValues actually work. I read this as 'it needs to be flatmap', but I expect that's untrue.

How about change k and _ into names of the things in those tuples (even though they are unused) to help me out.
(I'm not sure about that. Might end up cluttered. Maybe just initials like k could be pn, and list could be pnPoPairs.  The _ could be pn,)

Please also add the type to packageOwnersMap so I can at least see what this is trying to create.

Reply:
> Thoughts on changing this to something like:

So .mapValues iterates over values, but returns the entire map back, which is why I went with .map instead of .values.map which doesn't return the entire map.

I'll look into the RichMap conversion